pkg/mdlsub/subscriber_callback.go

```diff
@@ -44,7 +44,7 @@ func NewSubscriberCallbackFactory(transformers ModelTransformers, outputs Output
 		metricWriter := metric.NewDaemonWriter(defaultMetrics...)
 
 		callback := &SubscriberCallback{
-			logger:       logger,
+			logger:       logger.WithContext(ctx),
```

Comment:
```go
	logger := s.logger.WithContext(ctx).WithFields(log.Fields{
		"modelId": spec.ModelId,
		"type":    spec.CrudType,
		"version": spec.Version,
	})
```

Isn't this code in line 92 already adding the context to the logger? Why do we need to add this here, too?